### PR TITLE
header html & scss, addition of scss files

### PR DESCRIPTION
**Pull request by kintem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,23 @@
     <title>Kinte Matulyte</title>
 </head>
 <body>
+    <div class="nav-section">
+    <nav class="nav">
+            <ul class="nav-list">
```
Comment (joe-fol) [inline index.html:13]: is there a particular reason for using a list for your links? I'm not knocking your code just interested

File: stylesheets/abstracts/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&family=Open+Sans:wght@300&display=swap');
+
+// Fonts
+$heading-font: 'Montserrat', sans-serif;
+$body-font: 'Open Sans', sans-serif;
+
+// Colors
+$primary-color: #FFD271;
```
Comment (joe-fol) [inline stylesheets/abstracts/_variables.scss:8]: you could use comments beside the colors to pin point where there used e.g primary colour for nav bar

File: index.html (hunk 1 above)
Comment (kintem) [reply index.html:13]: good question, I think it's just force of habit at this point! I think pre-html5 using lists related to accessibility, but I don't think it matters as much anymore

File: stylesheets/abstracts/_variables.scss (hunk 2 above)
Comment (kintem) [reply stylesheets/abstracts/_variables.scss:8]: good shout! I think once I use the colours more across the rest of the site I'll definitely do that

File: index.html (hunk 1 above)
Comment (joe-fol) [reply index.html:13]: yeah i think i saw this maybe from free code camp a while back, for a minute i thought i had missed a part of a lesson.

**Pull request merged**